changelog/2024-11-25T20_38_27+08_00_reset_strategy

```diff
@@ -0,0 +1,2 @@
+CHANGED: `ResetStrategy` now contains the reset function/value, to avoid
+dummy arguments on `NoClearOnReset`
```

Comment:
```suggestion
CHANGED: `ResetStrategy` now contains the reset function, to avoid dummy arguments on `NoClearOnReset`
```

Since we now do `()` on `blockRam1`, I feel the message should omit _value_.

Incidentally, it wasn't something that would block approval and I would have happily merged it as-is, but ideally a changelog message is one line per entry (since this is one entry, it means all on one line) and should have a final newline. Since building the Changelog is still a manual process, it's all not super strict, but it's nice to have to reduce effort in building the real Changelog.